### PR TITLE
Fixed details link click and added DetailsText component.

### LINKED ISSUE #447: Volunteer can carry out group of missions
**Description**

As a volunteer, I can actually carry out a group of missions, viewing, and updating information as needed.

------------------------------
Tasks: 
- Create UI for displaying grouped missions
- Update Mission Card to show the new UI of how Pick-up/Drop-off is shown (2 columns)
- Create MissionGroup component, based on the group ID

[Airtable #122](https://airtable.com/tblRjd2McKgRW9MKj/viwlRRNI0PZIxrGkc/recFzfAMFuhOuhXRp?blocks=hide)

[Figma](https://www.figma.com/file/GLg0zPNnwMEGIb9TytCrxQ/Resilience-App?node-id=31996%3A17118)

### PR DESCRIPTION
**Pull request by laredotornado**
Fixes #447 

https://airtable.com/tblRjd2McKgRW9MKj/viwlRRNI0PZIxrGkc/recFzfAMFuhOuhXRp

### CODE REVIEW

File: src/app/dashboard/Missions/DetailsText.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from "@material-ui/core/styles";
+import React, { useEffect, useState } from "react";
+import _ from "../../utils/lodash";
+
+const FoodBoxDetails = ({ showType, details }) => {
+  return (
+    <>
+      {showType ? <b>Food Box</b> : null}
+      {_.get(details, "needs")?.map((box, index) => {
```
Comment (mat10tng) [inline src/app/dashboard/Missions/DetailsText.jsx:9]: yeah, sorry, the coding standard change on this, lets go with details?.needs

File: src/app/dashboard/Missions/DetailsText.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        return (
+          <div key={index}>
+            {_.get(box, "quantity")} x {_.get(box, "name")}
```
Comment (mat10tng) [inline src/app/dashboard/Missions/DetailsText.jsx:12]: box?.quantity box?.name

Comment (laredotornado) [reply src/app/dashboard/Missions/DetailsText.jsx]: K this has been corrected.

File: src/app/dashboard/Missions/DetailsText.jsx (hunk 1 above)
Comment (laredotornado) [reply src/app/dashboard/Missions/DetailsText.jsx]: You got it.  Updated.

**Pull request merged**